main.py

```diff
@@ -7,34 +7,60 @@
 
 class Shot(Image):
     velocity = Vector(0, 0)
+    shot_speed = 600
+    def clone(self):
```

Comment:
This creates a new shot object and copies all of the properties from _this_ shot.